parts/kubernetesagentresourcesvmas.t

```diff
@@ -189,14 +194,57 @@
       },
       "type": "Microsoft.Compute/virtualMachines"
     },
+    {{if UseManagedIdentity}}
     {
+      "apiVersion": "2014-10-01-preview",
+      "copy": {
+         "count": "[variables('{{.Name}}Count')]",
+         "name": "vmLoopNode"
+       },
+      "name": "[guid(concat('Microsoft.Compute/virtualMachines/', variables('{{.Name}}VMNamePrefix'), copyIndex(), 'vmidentity'))]",
+      "type": "Microsoft.Authorization/roleAssignments",
```

Comment:
Dunno why this showed in Reviewable for me to comment on but didn't post back here. Anyway, what I wrote:

>I'm not sure actually. I don't know if go tmpl has a generic include function or if we use it anywhere. I'd selfishly say I'd prefer to punt on it given my lack of immediate knowledge.